### PR TITLE
[strings] Improve Czech Android translations

### PR DESCRIPTION
**Pull request by Grreby**
Third attempt at trying to improve OM, haha. I'm sorry if I did it wrong again, but from what I've found, I should change this file. Also I'm not sure how to make strikethrough in .txt file.
Signed-off-by: Jan Podhorsky grreby@proton.me

### CODE REVIEW

**Review by rtsisyk — APPROVED** [review]

**Comment by rtsisyk** [comment]
@Grreby , you will be the first member of @organicmaps/translations-cs team. I invited you.

**Comment by rtsisyk** [comment]
Could you please add your real name to `Signed-off-by`?  We need something to identify you. 

https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md#legal-requirements

**Comment by Grreby** [comment]
> Could you please add your real name to `Signed-off-by`? We need something to identify you.
> 
> https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md#legal-requirements

Is it enough like that?

**Comment by rtsisyk** [comment]
If you can make `git commit --amend -s` + force-push (to have it in the commit message), it would be great!

**Comment by Grreby** [comment]
> If you can make `git commit --amend -s` + force-push (to have it in the commit message), it would be great!

I'm sorry, I don't have a computer with me rn, and I can't find any way to do this in github interface.

File: android/src/fdroid/play/listings/cs-CZ/full-description.txt [hunk 1]
```diff
@@ -30,23 +30,23 @@ Bez sledovacích zařízení a dalších špatných věcí:
...
 • Žádné shromažďování dat
-• Žádné telefonování domů
+• Žádné volání domů
 • Žádná obtěžující registrace
 • Žádné povinné výukové programy
 • Žádný hlučný e-mailový spam
 • Žádná push oznámení
 • Žádný crapware
-• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Čistě ekologické produkty
+• ~~Žádné pesticidy~~ Čistě organické
```
Comment (biodranik) [inline android/src/fdroid/play/listings/cs-CZ/full-description.txt:39]: Google Play does not support markdown.
```suggestion
• Ž̶á̶d̶n̶é̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶y̶ Čistě organické
```

**Review by biodranik — APPROVED** [review]

**Comment by biodranik** [comment]
That should be enough, thanks!

**Pull request merged**